pkg/shared/client.go

```diff
@@ -65,3 +71,20 @@ func getConfigFromServiceAccount() *rest.Config {
 	}
 	return config
 }
+
+// eventRecorder returns an EventRecorder type that can be
+// used to post Events to different object's lifecycles.
+func getEventRecorder() record.EventRecorder {
+
+	eventBroadcaster := record.NewBroadcaster()
```

Comment:
This doesn't imply any later object cleanup that we would need to do, does it?

Reply:
this doesn't create any object in kubernetes. so no need to free anything

Reply:
👍